x86_build.bat

```diff
@@ -0,0 +1,3 @@
+Add MD5 to the build etc...
```

Comment:
[0](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS126858)
A weak or broken hash algorithm was detected.
Suggested fixes:
+Add SHA256 to the build etc...
+Add SHA512 to the build etc...